admin/code/LeftAndMain.php

```diff
@@ -355,10 +356,12 @@ protected function getSchemaResponse($form) {
 	 * based on the request data.
 	 *
 	 * @param Form $form
+	 * @param String $id Optional, will default to the current request URL
 	 * @return array
 	 */
-	protected function getSchemaForForm(Form $form) {
+	protected function getSchemaForForm(Form $form, $id = null) {
 		$request = $this->getRequest();
+		$id = $id ? $id : $request->getURL();
```

Comment:
I was having some reservations about using the request URL for the "id" of the schema, perhaps `$id` could be mandatory.
`$id` may get removed in the future, so maybe not...
What are your thoughts about this? @chillu